book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @tittle = title
+    @author = author
+    @rentals = []
+  end

```

Comment:
- [ ] **Required:** This class should contain a method to add rentals. This class returns an instance of the Rental class.